tests/integration/golang/helpers/gs.go

```diff
@@ -19,3 +24,31 @@ func NewGSClient(endpoint string) (*storage.Client, error) {
 
 	return client, nil
 }
+
+// PrepareTestBuckets prepares test buckets for further usage in tests:
+// - delete provided buckets and all the content within them.
+// - create the same empty bucket.
+func PrepareTestBuckets(client *storage.Client, buckets []string) error {
```

Comment:
I think this should be split between delete (called during teardown) and create (called during setup)

Reply:
done.